app/models/user/profile.rb

```diff
@@ -20,4 +20,10 @@ def contributions_tab?
 
   memoize
   def badges_tab? = user.revealed_badges.exists?
+
+  memoize
+  def num_solutions_published = user.solutions.published.count
+
+  memoize
+  def num_testimonials = user.mentor_testimonials.published.count
```

Comment:
We could cache these on user_data too, but I'm working on the basis of one thing at a time!